indico/modules/designer/placeholders.py

```diff
@@ -297,3 +299,36 @@ class RegistrationTicketQRPlaceholder(Placeholder):
     @classmethod
     def render(cls, registration):
         return generate_ticket_qr_code(registration)
+
+class RegistrationFieldPlaceholder(ParametrizedPlaceholder):
```

Comment:
PEP8: there should be two empty lines between top-level classes (and functions)

Reply:
You might want to subclass `FieldPlaceholder`; like this you can skip almost all of your custom code in here.

Since you probably don't want to include all regform fields here (personal data is already handled with more explicit placeholder classes), you might want to move the `own_placeholder_types` set to the class level so you can override that in your subclass.